modules/ui/map3d_viewer.js

```diff
@@ -225,6 +226,7 @@ export function uiMap3dViewer(context) {
       if (d3_event) d3_event.preventDefault();
 
       _isHidden = !_isHidden;
```

Comment:
Avoid making a liar out of your variables whenever possible- as a reviewer, I can't tell if the viewer is hidden, or should be hidden, and whether it was already hidden prior to this line, or should be hidden after this line. 

Think in terms of 'desired' vs. 'actual' state. Try to have your var names make semantic sense everywhere they're seen, or their meaning becomes muddied. 

Try to refactor this method by using a local variable named `shouldHide`. Take action on `shouldHide`. Once you're done taking action, update the state of `_isHidden` by toggling it. This makes the intent of your code much clearer!

Reply:
Done. I used `shouldShow` since I don't have to invert `urlhash.getParam("map3d")` that way.